packages/aft/lib/src/commands/integration_tests_command.dart

```diff
@@ -0,0 +1,179 @@
+    );
+
+    final platformSelections = [
+      'VM',
+      'Web',
+    ];
+    final platformPrompt = Select(
+      prompt: 'Are you running on a VM or Web?',
+      options: platformSelections,
+    );
+
+    final deviceSelections = <String>[];
+    final devicePrompt = Select(
+      prompt: 'Select a device:',
+      options: deviceSelections,
+    );
+
+    final selectedPackage = packageSelections[packagePrompt.interact()];
+    final selectedPlatform = platformSelections[platformPrompt.interact()];
+
+    switch (selectedPlatform) {
+      case 'VM':
+        var devices = <ActiveDevice>[];
+        await startFlutterProcess(
+          [
+            'devices',
+            '--machine',
+          ],
+          selectedPackage,
+          printStream: false,
+        ).then((result) {
+          final list = json.decode(result) as Iterable;
+          devices = List<ActiveDevice>.from(
+            list.map(
+              (model) => ActiveDevice(
+                name: model['name'] as String,
+                id: model['id'] as String,
+              ),
+            ),
+          )..forEach((ActiveDevice el) {
+              if (el.name != 'Chrome') {
+                deviceSelections.add(el.name);
+              }
+            });
+        });
+
+        final devicePromptSelection = deviceSelections[devicePrompt.interact()];
+        final selectedDevice = devices
+            .firstWhere((device) => device.name == devicePromptSelection);
+
+        final results = await _executeTests(
+          [
+            'test',
+            'integration_test',
+            '-d',
+            selectedDevice.id,
+          ],
+          selectedPackage,
+          1,
+        );
+        stdout.write(results);
+        break;
+
+      // TODO(dnnoyes) get a parse driver results properly
+      case 'Web':
+        Process? process;
+        try {
+          process = await Process.start('chromedriver', ['--port=4444']);
+          stdout.write('Started chromedriver \n');
+        } on Exception catch (e) {
+          stderr
+            ..write('Tried and failed to start chromedriver.')
+            ..write(
+                'Make sure you are running chromedriver on port 4444 or that chromedriver is in your path.')
+            ..write(e);
+        }
+        final results = await _executeTests(
+          [
+            'drive',
+            '--driver=test_driver/integration_test.dart',
+            '--target=integration_test',
+            '-d',
+            'web-server',
+            '--dart-define=WEB_INTEG=true',
+          ],
+          selectedPackage,
+          2,
+        );
+        stdout.write(results);
+        process?.kill();
+        break;
+    }
+  }
+
+  Future<String> _executeTests(
+    List<String> args,
+    String package,
+    int fileArgIndex,
+  ) async {
+    final testResults = <String>['TEST RESULTS: '];
+    final files = await Directory('../../$package/example/integration_test')
```

Comment:
This would break depending on where you run `aft` from right? That's not the DX we want and `AmplifyCommand` has helpers for this already.

Reply:
Ah ok great, didn't see these helpers. Thanks.